user/validation/explicit/column-collapse.md

```diff
@@ -0,0 +1,75 @@
+# Column Collapse
+
+
+
+## Hydrostatic Analytical Solution
+
+Given the material points cannot move horizontally, the hydrostatic column collapse problem should yield hydrostatic condition. We expect no deviatoric stress component in hydrostatic condition and thus Bingham fluid model will give the same pressure for all normal stress directions ($\sigma_{xx}, \sigma_{yy}, \sigma_{zz}$).
```

Comment:
The hydrostatic validation involves applying gravity loading to a column of fluid restrained in both lateral directions and along the bottom plane. The Bingham fluid model will yield hydrostatic pressures for all normal stress directions ($\sigma_{xx}$, $\sigma_{yy}$, $\sigma_{zz}$).
```suggestion
Given the material points cannot move horizontally, the hydrostatic column collapse problem should yield hydrostatic condition. We expect no deviatoric stress component in hydrostatic condition and thus Bingham fluid model will give the same pressure for all normal stress directions ($\sigma_{xx}, \sigma_{yy}, \sigma_{zz}$).
```